charts/rancher-istio/0.0.1/charts/certmanager/values.yaml

```diff
@@ -4,12 +4,14 @@
 # gateway must be updated by adding 'secretVolumes'. After the gateway
 # restart, DestinationRules can be created using the ACME-signed certificates.
 enabled: false
+replicaCount: 1
 image:
   hub: quay.io
```

Comment:
should remove `image.hub` as we are using `rancher/jetstack-cert-manager-controller` from dockerhub, 
also change the `image: {{ .Values.image.hub }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}` to `image: {{ .Values.image.repository }}:{{ .Values.image.tag }}` in certmanager/templates/deployment.yaml  Line 41